src/runtime/utils/index.ts

```diff
@@ -91,3 +91,10 @@ export function parseSize (input: string | number | undefined = '') {
     }
   }
 }
+
+export function parseDensities (input: string | undefined = '') {
```

Comment:
should be more strictly typed to return `number[]`

How to deal with invalid input format, maybe it should throw/print some WARN messages and return `[]` if unable to parse...